testing/test_terminal.py

```diff
@@ -718,3 +719,63 @@ def pytest_terminal_summary(terminalreporter):
         *==== hello ====*
         world
     """)
+
+@pytest.mark.parametrize("exp_color, exp_line, stats_arg", [
+    # The method under test only cares about the length of each
+    # dict value, not the actual contents, so tuples of anything
+    # suffice
+
+    # Important statuses -- the highest priority of these always wins
+    ("red",    "1 failed",               {"failed":     (1,)}),
```

Comment:
Re. the spacing, you mean?  Will do. (A previous change of mine was accepted looking like this, so I did the same this time.)